### PR TITLE
The value of Configuration-Structure-Schema ASN.1

### PR DESCRIPTION
**Pull request by changab**
- The size of UPER binary 100 bytes
- Some minor changes on schema.asn.

### CODE REVIEW

File: binary/asn1/schema.asn [hunk 1]
```diff
@@ -33,7 +33,7 @@ Configuration-Structure-Schema DEFINITIONS AUTOMATIC TAGS ::= BEGIN
       ...
    }
    Configuration ::= SEQUENCE {
-      harts SEQUENCE OF Hart OPTIONAL,
+      harts SEQUENCE (SIZE(1..64)) OF Hart OPTIONAL,
```
Comment (timsifive) [inline binary/asn1/schema.asn:36]: I don't think we can limit the size like that. E.g. Esperanto has a core with more than 1000 harts already.

File: binary/asn1/schema.asn.value [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                  matchEqual TRUE,
+                  matchNapot TRUE,
+                  matchGreaterEqual TRUE,
+                  matchLess TRUE,
+                  matchLowMask TRUE,
+                  matchHighMask TRUE,
+                  matchNotEqual TRUE,
+                  matchNotNapot TRUE,
+                  matchNotLowMask TRUE,
+                  matchNotHighMask TRUE,
```
Comment (timsifive) [inline binary/asn1/schema.asn.value:38]: In the other example these are false, and some others are as well.

File: binary/asn1/schema.asn.value [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                  timingBefore FALSE,
+                  timingAfter FALSE,
+                  sizeAny FALSE,
+                  sizeS8 FALSE,
+                  sizeS16 FALSE,
+                  sizeS32 FALSE,
+                  sizeS64 FALSE,
+                  sizeS80 FALSE,
+                  sizeS96 FALSE,
+                  sizeS112 FALSE,
```
Comment (timsifive) [inline binary/asn1/schema.asn.value:65]: In this format is it allowed to omit default (false) values? That would be really helpful for users writing descriptions.

File: binary/asn1/schema.asn (hunk 1 above)
Comment (changab) [reply binary/asn1/schema.asn:36]: This is just the experiment of using SIZE syntax to see if we would get the reduced size. The entire schema should still have room to tailor for the concise size.

File: binary/asn1/schema.asn.value (hunk 2 above)
Comment (changab) [reply binary/asn1/schema.asn.value:38]: I will check it over again.

File: binary/asn1/schema.asn.value (hunk 3 above)
Comment (changab) [reply binary/asn1/schema.asn.value:65]: Hmm, I remember there is something like Default syntax, not sure how to use it and what is the advantage. I will check it.

**Pull request closed without merging**